addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -228,7 +228,7 @@ def test_channel_command_help_in_livechat(self):
                     "type": "discuss.channel/transient_message",
                     "payload": {
                         "body":
-                            "<span class='o_mail_notification'>You are in a private conversation with <b>@Paul</b> and <b>@Visitor</b>."
+                            f"<span class='o_mail_notification'>You are in a private conversation with <b>@{self.operators[1].partner_id.name}</b> and <b>@Visitor</b>."
```

Comment:
Rather than making the name dynamic, we should ensure it takes the one we expect

Reply:
This was not problematic in the test, I just made the change to all those non-explicated data so that it is related to the test correctly. But for this line, if wanted, it can be reverted.